VaneVC88/procedural.js

```diff
@@ -0,0 +1,20 @@
+function isPrime(num) {
+  for(var j = 2; j <= num/2; j++){
+    if(num % j == 0){
+      isPrime = false;
+    }
+  }
+}
+
+function getPrimes(start, end) {
+  for (var i = start; i < end; i++) {
+    var isPrime = true;
+    var arreglo = [];
```

Comment:
este arreglo debería estar definido afuera del `for` ya que por cada iteración de este, inicializara un arreglo vacío.